Formula/cgal.rb

```diff
@@ -16,48 +16,60 @@ class Cgal < Formula
+      -DWITH_CGAL_Qt5=ON
     ]
 
     system "cmake", ".", *args
     system "make", "install"
   end
-
   test do
-    # https://doc.cgal.org/latest/Algebraic_foundations/Algebraic_foundations_2interoperable_8cpp-example.html
+    # https://doc.cgal.org/latest/Triangulation_2/Triangulation_2_2draw_triangulation_2_8cpp-example.html and  https://doc.cgal.org/latest/Algebraic_foundations/Algebraic_foundations_2interoperable_8cpp-example.html
     (testpath/"surprise.cpp").write <<~EOS
+      #include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+      #include <CGAL/Triangulation_2.h>
+      #include <CGAL/draw_triangulation_2.h>
       #include <CGAL/basic.h>
       #include <CGAL/Coercion_traits.h>
       #include <CGAL/IO/io.h>
+      #include <fstream>
+      typedef CGAL::Exact_predicates_inexact_constructions_kernel K;
+      typedef CGAL::Triangulation_2<K>                            Triangulation;
+      typedef Triangulation::Point                                Point;
+ 
       template <typename A, typename B>
-      typename CGAL::Coercion_traits<A,B>::Type
-      binary_func(const A& a , const B& b){
-          typedef CGAL::Coercion_traits<A,B> CT;
-          CGAL_static_assertion((CT::Are_explicit_interoperable::value));
-          typename CT::Cast cast;
+      typename CGAL::Coercion_traits<A,B>::Type	      
+      binary_func(const A& a , const B& b){	      
+          typedef CGAL::Coercion_traits<A,B> CT;	      
+          CGAL_static_assertion((CT::Are_explicit_interoperable::value));	      
+          typename CT::Cast cast;	        
           return cast(a)*cast(b);
       }
-      int main(){
-          std::cout<< binary_func(double(3), int(5)) << std::endl;
-          std::cout<< binary_func(int(3), double(5)) << std::endl;
-          return 0;
-      }
+
+      int main() {
+        std::cout<< binary_func(double(3), int(5)) << std::endl;
+        std::cout<< binary_func(int(3), double(5)) << std::endl;
+        std::ifstream in("data/triangulation_prog1.cin");
+        std::istream_iterator<Point> begin(in);
+        std::istream_iterator<Point> end;
+        Triangulation t;
+        t.insert(begin, end);
+        CGAL::draw(t);
+        return EXIT_SUCCESS;
+       }
     EOS
     (testpath/"CMakeLists.txt").write <<~EOS
-      cmake_minimum_required(VERSION 3.1...3.13)
-      find_package(CGAL)
+      cmake_minimum_required(VERSION 3.1...3.15)
+      find_package(CGAL COMPONENTS Qt5)
+      add_definitions(-DCGAL_USE_BASIC_VIEWER -DQT_NO_KEYWORDS)
       add_executable(surprise surprise.cpp)
-      target_link_libraries(surprise PRIVATE CGAL::CGAL)
+      target_link_libraries(surprise PUBLIC CGAL::CGAL_Qt5)
     EOS
-    system "cmake", "-L", "-DCMAKE_BUILD_RPATH=#{HOMEBREW_PREFIX}/lib", "-DCMAKE_PREFIX_PATH=#{prefix}", "."
+    system "cmake", "-L", "-DQt5_DIR=#{HOMEBREW_PREFIX}/opt/qt/lib/cmake/Qt5", "-DCMAKE_BUILD_RPATH=#{HOMEBREW_PREFIX}/lib", "-DCMAKE_PREFIX_PATH=#{prefix}", "."
```

Comment:
```suggestion
    system "cmake", "-L", "-DQt5_DIR=#{Formula["qt"].opt_lib}/cmake/Qt5", "-DCMAKE_BUILD_RPATH=#{HOMEBREW_PREFIX}/lib", "-DCMAKE_PREFIX_PATH=#{prefix}", "."
```

Reply:
I have implemented that change in https://github.com/Homebrew/homebrew-core/pull/52462/commits/0354af31ca9ae299e20a7cd84f25b32bcc4edfff.

Thanks for the suggestion.